Makefile

```diff
@@ -189,3 +193,25 @@ wordpress/.env:
 wordpress/web/app/uploads/2022:
 	@echo '==> Downloading wordpress uploads'
 	./devTools/docker/download-wordpress-uploads.sh
+
+deploy:
+	@echo '==> Starting from a clean slate...'
+	rm -rf itsJustJavascript
+	
+	@echo '==> Building...'
+	yarn
+	yarn run tsc -b
+	
+	@echo '==> Deploying...'
+	yarn buildAndDeploySite live
+
+stage:
+	@echo '==> Starting from a clean slate...'
+	rm -rf itsJustJavascript
+	
+	@echo '==> Building...'
+	yarn
+	yarn run tsc -b
+	
+	@echo '==> Deploying to staging...'
+	yarn buildAndDeploySite staging
```

Comment:
In this case the `-clean` is redundant after removing the `itsJustJavascript` directory, we could update that command too.

Otherwise, it's pretty much the same, except that I also run `yarn`, which covers the case where you have changed branch and package dependencies have changed.

Locally, I run this script: https://github.com/owid/lars-grapher-scripts/blob/master/grapher-deploy#L15 

It speaks out loud to me to say if the deploy succeed or failed, which is nice and lets you minimise the tab and keep moving.